tools/license/check-license.sh

```diff
@@ -21,7 +21,8 @@ function files {
 		-not -name "OTFormat.swift" \
 		-not -name "OTTracer.swift" \
 		-not -name "OTReference.swift" \
-		-not -name "OTSpanContext.swift"
+		-not -name "OTSpanContext.swift" \
+        -not -name "Versioning.swift"
```

Comment:
```suggestion
		-not -name "Versioning.swift"
```

Reply:
ups, it looked well tabulated on my editor